packages/main/src/CardHeader.js

```diff
@@ -0,0 +1,302 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import { isSpace, isEnter } from "@ui5/webcomponents-base/dist/Keys.js";
+import CardHeaderTemplate from "./generated/templates/CardHeaderTemplate.lit.js";
+import Icon from "./Icon.js";
+
+import {
+	AVATAR_TOOLTIP,
+	ARIA_ROLEDESCRIPTION_CARD_HEADER,
+	ARIA_ROLEDESCRIPTION_INTERACTIVE_CARD_HEADER,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import cardHeaderCss from "./generated/themes/CardHeader.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-card-header",
+	languageAware: true,
+	managedSlots: true,
+	slots: /** @lends sap.ui.webcomponents.main.CardHeader.prototype */ {
+		avatar: {
+			type: HTMLElement,
+		},
+
+		/**
+		 * Defines an action, displayed in the right most part of the header.
+		 * <br><br>
+		 * <b>Note:</b> If set, the <code>status</code> text will not be displayed,
+		 * you can either have <code>action</code>, or <code>status</code>.
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 * @since 1.0.0-rc.15
+		 */
+		action: {
+			type: HTMLElement,
+		},
+	},
+	properties: /** @lends sap.ui.webcomponents.main.CardHeader.prototype */ {
+
+		/**
+		 * Defines the title displayed in the component header.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.0.0-rc.15
```

Comment:
You need to only add since tag to the class itself, as it is a new component, we don't need to since tag for each prop or slot.